src/main/java/org/jenkinsci/plugins/github/webhook/WebhookManager.java

```diff
@@ -183,7 +185,8 @@ public GHHook apply(GitHubRepositoryName name) {
         return new Predicate<GHHook>() {
             @Override
             public boolean apply(GHHook input) {
-                LOGGER.debug(format("%s {} (events: {})", format), input.getUrl(), input.getEvents());
+                final GHHook _input = Validate.notNull(input);
```

Comment:
please don't use underline in var names. What for you reassign this to final var? notNull returns input without any change. And nothing can change input to make it final. Don't fix all FB warnings.